module/models/action.mjs

```diff
@@ -1417,9 +1423,12 @@ export default class CrucibleAction extends foundry.abstract.DataModel {
         await test.confirm.call(this, reverse);
       } catch(err) {
         console.error(new Error(`"${this.id}" action confirmation failed`, {cause: err}));
+        return false;
       }
     }
 
+    const isNegated = this.message?.getFlag("crucible", "isCounterspelled");
+
     // Additional Actor-specific consequences
     this.actor.onDealDamage(this, this.outcomes);
```

Comment:
An action being negated should probably prevent it from triggering `onDealDamage` workflows.